lib/urbanopt/geojson/schema/electrical_connector_properties.json

```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "id": "http://json-schema.org/openstudio-urban-modeling/district_system_properties.json#",
-  "title": "URBANopt Electrical Connector",
-  "description": "Schema for an URBANopt Electrical Connector object",
+  "title": "URBANopt Electrical Line",
```

Comment:
What was the reason for renaming all 'connector' to 'line'?

Reply:
Mostly for consistency with electrical model naming conventions.